tensordata/serology.py

```diff
@@ -67,16 +62,16 @@ def sharedElements(occurence: int, *args):
 
 
 def concat4D():
-    M = MGH4D(xarray=True).assign_coords(M_dict)
+    M = MGH4D().assign_coords(M_dict)
```

Comment:
After adding the functional data this cannot pass the test. Some changes need to be made here.